src/pages/contacts-page/contacts-edit/contacts-edit.js

```diff
@@ -0,0 +1,370 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
I think it makes sense to create one dumb component for the contact form and via props configure it to perform the creating or editing process of the contact. I will eliminate unnecessary duplication of logic